reviewers.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict
+import operator
+import os
+import re
+
+
+def prompt_for_user():
+    while True:
+        try:
+            user_input = input("\nName or email (case insensitive): ") 
```

Comment:
nit: Since we're just checking the prefix of the name (or email), not sure if we should mention that here.